test/extended/util/client.go

```diff
@@ -105,8 +106,17 @@ func NewCLIWithFramework(kubeFramework *framework.Framework) *CLI {
 	return cli
 }
 
+// NewCLIWithPodSecurityEnforceLevel initializes the CLI the same way as `NewCLI()`
+// but the given pod security level is applied to the created e2e test namespace.
+func NewCLIWithPodSecurityEnforceLevel(project string, level admissionapi.Level) *CLI {
```

Comment:
this should now probably be called just `NewCLIWithPodSecurityLevel` (remove the `Enforce` from the name)

Reply:
fixed